<launch>

    <arg name="device" default = "17221121"/>
    <arg name="rate" default="15"/>

    <node name="single_reader" pkg="ptgrey_reader" type="single_reader"
         ns="pg_$(arg device)" output="screen">
        <remap from="/image_out" to="image_raw" />
        <param name="is_pub" type="bool" value="true"/>
        <param name="is_show" type="bool" value="false"/>
        <param name="is_print" type="bool" value="true"/>
        <param name="serialNum" type="int" value="$(arg device)"/>
        <param name="frameRate" type="double" value="$(arg rate)"/>
        <param name="is_auto_shutter" type="bool" value="false"/>
        <param name="is_sync" type="bool" value="false"/>
        <param name="shutter" type="double" value="5.0"/>
        <param name="brightness" type="double" value="0.1"/>
        <param name="exposure" type="double" value="0.1"/>
        <param name="gain" type="double" value="1.0"/>
    </node>



</launch>
